nixos/modules/misc/man-db.nix

```diff
@@ -52,9 +52,13 @@ in
     environment.systemPackages = [ cfg.package ];
     environment.etc."man_db.conf".text =
       let
+        mandbForBuild = if pkgs.stdenv.hostPlatform == pkgs.stdenv.buildPlatform then
```

Comment:
Wouldn't it be easier to use nativeBuildInputs here?

Reply:
Unfortunately no.

Reply:
I'm sorry I had forgotten that pkgs was changed to __splicedPackages in the modules so now nativeBuildInputs works properly in nixos

https://github.com/NixOS/nixpkgs/commit/f7077ba1311a49c91493a6696f4ec0dd568f297a

So yes what Sandro suggested will now work 😄

Reply:
Does this need callPackage though?

Reply:
> Does this need callPackage though?

nope because the thing that makes callPackage needed for cross-compile is splicedPackages which can be worked around by manually passing __splicedPackages like now